docs/installation/linux/debian.md

```diff
@@ -77,7 +77,7 @@ from the new repository:
 
  5. Add the new `GPG` key.
 
-         $ apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+         $ apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 58118 E89F3A912897C070ADBF76221572C52609D
```

Comment:
Also, removing `p80.` - would that open the possibility that a mirror is used that is **not** running on port 80 (e.g., only on 443)?

/cc @tianon